### DISCUSSION TITLE
Driver 1.1.1 isn't available in maven repo for pull from gradle build

### DISCUSSION THREAD

**Discussion by abedali** [awslabs/aws-mysql-jdbc#289] [Q&A] [answered]
Hi,

Thank you for the new version of the driver 1.1.1 release (9/22).
Issue: It's not available for pull using gradle build, looks like it's not published in maven yet.
Can you please help?

Could not resolve all files for configuration ':compileClasspath'.
   > Could not find software.aws.rds:aws-mysql-jdbc:1.1.1.

/Abed

**Comment by hsuamz** [comment]
Hello @abedali,

We're preparing for the 1.1.1 release and are in the process of verification. We'll make the release public once we have confirmation.

Thanks,

Matthew

**Comment by hsuamz** [comment, marked as answer]
Hello @abedali,

The release is now public. You should have no issue in retrieving the latest release.

Thanks,

Matthew

**Reply by abedali** [reply]
Thanks @hsuamz , I'm good now.
